Source/JavaScriptCore/runtime/SymbolConstructor.cpp

```diff
@@ -70,6 +70,9 @@ void SymbolConstructor::finishCreation(VM& vm, SymbolPrototype* prototype)
     putDirectWithoutTransition(vm, vm.propertyNames->prototype, prototype, PropertyAttribute::DontEnum | PropertyAttribute::DontDelete | PropertyAttribute::ReadOnly);
 
     JSC_COMMON_PRIVATE_IDENTIFIERS_EACH_WELL_KNOWN_SYMBOL(INITIALIZE_WELL_KNOWN_SYMBOLS)
+    if (Options::useExplicitResourceManagement()) {
+        JSC_COMMON_PRIVATE_IDENTIFIERS_EACH_EXPLICIT_RESOURCE_MANAGEMENT_WELL_KNOWN_SYMBOL(INITIALIZE_WELL_KNOWN_SYMBOLS)
```

Comment:
We need to put `{` here because following macro will be expanded to two lines code